neurots/generate/orientations.py

```diff
@@ -360,3 +459,190 @@ def compute_interval_n_tree(soma, n_trees, rng=np.random):
         interval_n_trees = np.array([n_trees])
 
     return phi_intervals, interval_n_trees
+
+
+def sample_spherical_unit_vectors(rng):
+    """Sample a point uniformly on the sphere.
+
+    Args:
+        rng: random number generator
+    """
+    x = rng.normal(0, 1, 3)
+    return x / np.linalg.norm(x)
+
+
+def spherical_angles_to_pia_orientations(phis, thetas):
+    """Compute orientation from spherical angles where thetas are wirt to pia at [0, 1, 0].
+
+    Args:
+        phis (numpy.ndarray): Polar angles.
+        thetas (numpy.ndarray): Azimuthal angles.
+
+    Returns:
+        numpy.ndarray: The orientation vectors where each row correspnds to a phi-theta pair.
+    """
+    assert PIA_DIRECTION == [0, 1, 0], "Global pia direction is not compatible"
```

Comment:
This is only possible if the user deliberately updates the `neurots.generate.utils.PIA_DIRECTION`, right? In that case I'm not sure we have to check the value here. Maybe we could just ensure that it is not possible to update this value?

Reply:
yes, I wasn't sure what's best here, @lida?